design-library/source/_patterns/01-atoms/07-section-content/section-content.json

```diff
@@ -1,3 +1,4 @@
 {
-  "content": "<ul><li><a href=\"\">Tech</a></li><li><a href=\"\">News</a></li><li><a href=\"\">Tutorials</a></li><li><a href=\"\">Sports</a></li><ul>"
```

Comment:
Originally, this pattern was a section on the original sidebar. This pattern may change, but section content could vary depending on what is displayed in the section. Do you think we should keep this generic? If not, should we rename this pattern? It looks currently to be a blog tag, which is another existing pattern.